canonical/cn.ubuntu.com
# WD-11512 Refresh /download/risc-v page

## Done

Webpage refresh for the page /download/risc-v
## QA
- Check out this feature branch
- Run the site using the command dotrun
- View the site locally in your web browser at: http://0.0.0.0:8010/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- [Copydoc](https://docs.google.com/document/d/1SPeefGBJBh99XWEx0newZHPxp56mZuaElq1YtUzfY4M/edit)
- [Demo](https://cn-ubuntu-com-725.demos.haus/download/risc-v)
## Issue / Card
[WD-11512](https://warthogs.atlassian.net/browse/WD-11512)

[WD-11512]: https://warthogs.atlassian.net/browse/WD-11512?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
